devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -140,15 +142,37 @@ private createMeterEventMap(cmd) {
 	eventMap
 }
 
+// cmd.endPoints includes the USB ports but we don't want to expose them as child devices since they cannot be controlled so hardcode to just include the outlets
 def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelEndPointReport cmd, ep = null) {
 	if(!childDevices) {
-		addChildSwitches(cmd.endPoints)
+		if (isZoozZenStripV2()) {
+			addChildSwitches(5)
+		} else if (isZoozDoublePlug()) {
```

Comment:
Does Double Plug report wrong number of endpoints that this exception here is necessary?

Reply:
" // cmd.endPoints includes the USB ports but we don't want to expose them as child devices since they cannot be controlled so hardcode to just include the outlets "